src/patternfly/components/Truncate/examples/Truncate.md

```diff
@@ -8,44 +8,83 @@ import './Truncate.css'
 
 ## Examples
 
-### Notes
-The truncate component contains two child elements, `.pf-v6-c-truncate__start` and `.pf-v6-c-truncate__end`. If both `start` and `end` are present within `.pf-v6-c-truncate`, trucation will occur in the middle of the string. If only `.pf-v6-c-truncate__start` is present, truncation will occur at the end of the string. If only `.pf-v6-c-truncate__end` is present, truncation will occur at the beginning of the string. A `.pf-v6-c-popover` will be automatically applied to the PatternFly React implementation. `&lrm;` must be included at the end of string to denote the ending punctuation mark. Otherwise it will occur and the beggining of truncation for a `pf-v6-c-truncate__end` element.
+The default behavior of the `Truncate` component is to truncate based on whether the content can fit within the width of its parent container, and to prevent text from wrapping. The following examples that use this default behavior render the `Truncate` component inside a resizable container, allowing you to see how the parent container width affects the truncation.
```

Comment:
I could be mistaken, but I think if we're referring to a component name, it's lower-case and a regular word. Do you know of a different standard? 

```suggestion
The default behavior of the truncate component is to truncate based on whether the content can fit within the width of its parent container, and to prevent text from wrapping. The following examples that use this default behavior render the truncate component inside a resizable container, allowing you to see how the parent container width affects the truncation.
```

Reply:
I think it's what I'm used to when it comes to React example descriptions. cc @edonehoo thoughts/opinions?

Reply:
Thanks @edonehoo! We used to title case component names at some point, but it was inconsistent, and not for any particular reason. It was just how some folks chose to write it. A while back content did an audit and created some standards, then we did a sweep through the content and updated everything to be lower case, so I'd say go with lower case "truncate" like in my comment above 👍